examples/component/http-keyvalue-crud/wasmcloud.toml

```diff
@@ -0,0 +1,8 @@
+name = "crud"
+language = "tinygo"
+type = "component"
+version = "0.1.0"
+
+[component]
+wit_world = "hello"
```

Comment:
We haven't settled on a specific pattern here, but it might be nice to call this world `component`